### PR TITLE
add performance test for sparse tensor joins

### PR DESCRIPTION
**Pull request by arnej27959**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

@geirst please review
@havardpe @toregge FYI

### CODE REVIEW

File: tests/performance/sparse_tensors/sparse_global_tensor_join.rb [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+      threads_per_search(num_threads).
+      container(Container.new('mycc').
+                jvmargs('-Xms2g -Xmx2g').
+                search(searching).
+                docproc(DocumentProcessing.new)).
+      indexing('mycc')
+    return app
+  end
+
+  def test_sparse_tensors_stream_ranking
```
Comment (geirst) [inline tests/performance/sparse_tensors/sparse_global_tensor_join.rb:29]: Please rename test.

File: tests/performance/sparse_tensors/sparse_global_tensor_join.rb [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    set_description('Test performance of sparse tensor joins')
```
Comment (geirst) [inline tests/performance/sparse_tensors/sparse_global_tensor_join.rb:30]: Please add owner as well.

File: tests/performance/sparse_tensors/globaltensors.sd [hunk 3]
```diff
@@ -0,0 +1,17 @@
+search globaltensors {
+    document globaltensors {
+        field ident type string {
```
Comment (geirst) [inline tests/performance/sparse_tensors/globaltensors.sd:3]: This doesn't seem to be used, consider removing.

**Review by geirst — APPROVED** [review]
👍

**Comment by arnej27959** [comment]
@geirst PTAL

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**